tasks/rollbar-source-map.js

```diff
@@ -109,15 +108,11 @@ async function uploadAppVersions(config, rev) {
   await Promise.all(
     buildLogFiles.map(async file => {
       try {
-        const fd = await fs.openAsync(file, 'r');
-        // Read just the first 64 chars from the log file
-        const buff = new Buffer(64);
-        const read = await fs.readAsync(fd, buff, 0, 64, 0);
-        const header = buff.toString('utf-8', 0, read);
-        // Look for the webpack hash
-        const parsedHeader = header.match(/Hash:\s(\w+)/i);
-        if (parsedHeader && parsedHeader[1]) {
-          hashes.push(parsedHeader[1]);
+        const buffer = await fs.readFileAsync(file);
```

Comment:
Relevant changes, others are Prettier fixes

Reply:
The reason we were reading only a few chars here before was that we used to work with the large webpack stat report files. If the new webpack plugin generates small-ish files then the change is ok 👍

Reply:
Ah, I see, makes sense